TopPlaces/TopPlaces/Model/Flickr/FlickrPlaceInfo.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2016 Lightricks. All rights reserved.
+// Created by Gennadi Iosad.
+
+#import "PlacesPhotosProvider.h"
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FlickrPlaceInfo : NSObject<PlaceInfo>
```

Comment:
1) <code>NSObject &lt;PlaceInfo&gt;</code> (add space).
2) Since you declare that you implement the protocol <code>PlaceInfo</code> you don't need to specifically redeclare each method / property of it.